CHANGELOG.md

```diff
@@ -8,6 +8,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [Unreleased]
 
 ### Added
+- Implemented `FreqRange` utility class for frequency/wavelength handling with constructor methods `from_freq_interval()`, `from_wavelength()`, and `from_wvl_interval()`.   
+
+## [0.1.0] - 2025-07-02
```

Comment:
remove. And you can just put your bullet point under the existing ### Added section in the chanbgelog along with all of the most recent added items, eg here

https://github.com/flexcompute/tidy3d/blob/e11b187ba1693654e97f70ba007a96515044d317/CHANGELOG.md?plain=1#L16

might be easiest to just restore the state of `CHANGELOG` in develop and add the single line.